mozetl/topline/topline_schema.json

```diff
@@ -0,0 +1,109 @@
+{
+  "fields":[
+    {
+      "metadata":{
+
+      },
+      "name":"geo",
+      "nullable":true,
+      "type":"string"
+    },
+    {
+      "metadata":{
+
+      },
+      "name":"channel",
+      "nullable":true,
+      "type":"string"
+    },
+    {
+      "metadata":{
+
+      },
+      "name":"os",
+      "nullable":true,
+      "type":"string"
+    },
+    {
+      "metadata":{
+
+      },
+      "name":"hours",
+      "nullable":true,
+      "type":"double"
+    },
+    {
+      "metadata":{
+
+      },
+      "name":"crashes",
+      "nullable":true,
+      "type":"long"
+    },
+    {
+      "metadata":{
+
+      },
+      "name":"google",
+      "nullable":true,
+      "type":"long"
+    },
+    {
+      "metadata":{
+
+      },
+      "name":"bing",
+      "nullable":true,
+      "type":"long"
+    },
+    {
+      "metadata":{
+
+      },
+      "name":"yahoo",
+      "nullable":true,
+      "type":"long"
+    },
+    {
+      "metadata":{
+
+      },
+      "name":"other",
+      "nullable":true,
+      "type":"long"
+    },
+    {
+      "metadata":{
+
+      },
+      "name":"actives",
+      "nullable":true,
+      "type":"long"
+    },
+    {
+      "metadata":{
+
+      },
+      "name":"new_records",
+      "nullable":true,
+      "type":"long"
+    },
+    {
+      "metadata":{
+
+      },
+      "name":"default",
+      "nullable":true,
+      "type":"long"
+    },
+    {
+      "metadata":{
+
+      },
+      "name":"report_start",
+      "nullable":true,
+      "type":"integer"
```

Comment:
I'm pretty sure this should be a string field. MainSummaryView writes out the partition column in the same way, but `submission_date_s3` doesn't end up being read as an integer. I'm going to backfill the weekly topline data from airflow to double check this.